net/pdnsd/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2019 Xiaobo Tian
+# Copyright (C) 2001-2001 Thomas Moestl
+# Copyright (C) 2002-2010 Paul A. Rombouts
```

Comment:
nitpick, copyright lines seem to have a convention to order by years.

Reply:
Just a maintainer's change to the GPL